tavern/_plugins/rest/request.py

```diff
@@ -86,7 +87,7 @@ def get_request_args(rspec, test_block_config):
             rspec["headers"] = {i: j for i, j in headers.items() if i.lower() != "content-type"}
     elif headers:
         # This should only be hit if we aren't sending a file
-        if not has_content_header:
+        if not has_content_header and not rspec.get("stream"):
```

Comment:
I don't understand why this check is here - the streaming should affect the response, not the request content-type header